shogun-admin/modelconfigs/layer.json

```diff
@@ -36,6 +36,17 @@
   "entityName": "#i18n.entityName",
   "navigationTitle": "#i18n.navigationTitle",
   "subTitle": "",
+  "defaultEntity":{
+    "type": "TILEWMS",
+    "clientConfig" : {
+        "hoverable":false
+      },
+     "sourceConfig": {
+      "url": "https://ows.terrestris.de/osm/service?",
+      "layerNames": "OSM-WMS",
+      "useBearerToken": false
+     }
+  },
```

Comment:
Please recheck the indentation of the added json block.

Reply:
done in [a9ae915](https://github.com/terrestris/shogun-docker/pull/95/commits/a9ae915849471e589c8a1bab4121a8cc6b1ba39f)